Content/Archaeology/ArchaeologyManager.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using StarlightRiver.Content.GUI;
+using StarlightRiver.Core;
+using StarlightRiver.Packets;
+using StarlightRiver.Helpers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Terraria;
+using Terraria.DataStructures;
+using Terraria.GameInput;
+using Terraria.ID;
+using Terraria.ModLoader;
+using Terraria.ModLoader.IO;
+using Terraria.Map;
+using Terraria.UI;
+
+namespace StarlightRiver.Content.Archaeology
+{
+    public class ArchaeologyManager : ModSystem
+    { 
+        public override void Load()
+        {
+            On.Terraria.Main.DrawTiles += DrawArtifacts;
+        }
+
+        public override void Unload()
+        {
+            On.Terraria.Main.DrawTiles -= DrawArtifacts;
+        }
+
+        public override void PreUpdateDusts()
+        {
+            foreach (var item in TileEntity.ByID)
+            {
+                if (item.Value is Artifact artifact && artifact.IsOnScreen())
```

Comment:
single line conditionals should be inlined

Reply:
Fixed